.github/actions/cdn_code_deploy/action.yaml

```diff
@@ -0,0 +1,86 @@
+name: CDN Code Deploy 
+description: Action that Deploy an artifact to the CDN
+
+# This action allows you to deploy the passed artifact to the CDN.
```

Comment:
Do we need a specific action for artifact deployments? Can't be the same of a generic file deployment? What code changes?